src/common/Footer.jsx

```diff
@@ -1,7 +1,119 @@
 import React from 'react';
+import { Box, Grid, Typography, Link, makeStyles } from '@material-ui/core';
 
-const Footer = () => {
-    return (<h1>This is Footer.</h1>)
-}
+const useStyles = makeStyles((theme) => ({
+    root: {
+      flexGrow: 1,
+      paddingTop: theme.spacing(4),
+      paddingRight: theme.spacing(2),
+      paddingBottom: theme.spacing(10), 
+      paddingLeft: theme.spacing(2),
+      textAlign: 'center',
+      backgroundColor: '#34495E', 
+      color: '#FFFFFF', 
+    },
+    title: {
+        fontweight: 400, 
+    }, 
+    listOfItems: {
+        display: 'flex',
+        flexDirection: 'column', 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        flexDirection: "column", 
```